frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -834,6 +834,7 @@ export const LightThemeDateFilterRange = {
     await userEvent.click(filter);
 
     const popover = getLastPopover();
+    await userEvent.click(popover.getByRole("button", { name: "Add time" }));
```

Comment:
This add the time input to the snapshot, so we tested that the time input is properly theme as well.